From f684a3b819efa39b867c7fe219264716470bd8c6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 8 Aug 2010 14:21:43 +0000
Subject: [PATCH v2 4/4] t/README: Add SMOKE_{COMMENT,TAGS}= to smoke_report target
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The smoke server supports a free form text field with comments about a
report, and a comma delimited list of tags. Change the smoke_report
target to expose this functionality. Now smokers can send more data
that explains and categorizes the reports they're submitting.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/Makefile |    6 ++++++
 t/README   |    7 +++++++
 2 files changed, 13 insertions(+), 0 deletions(-)

diff --git a/t/Makefile b/t/Makefile
index 00f6482..b81fb91 100644
--- a/t/Makefile
+++ b/t/Makefile
@@ -69,6 +69,12 @@ SMOKE_UPLOAD_FLAGS =
 ifdef SMOKE_USERNAME
 	SMOKE_UPLOAD_FLAGS += -F username="$(SMOKE_USERNAME)" -F password="$(SMOKE_PASSWORD)"
 endif
+ifdef SMOKE_COMMENT
+	SMOKE_UPLOAD_FLAGS += -F comments="$(SMOKE_COMMENT)"
+endif
+ifdef SMOKE_TAGS
+	SMOKE_UPLOAD_FLAGS += -F tags="$(SMOKE_TAGS)"
+endif
 
 smoke_report: smoke
 	curl \
diff --git a/t/README b/t/README
index 69af05f..9f676d7 100644
--- a/t/README
+++ b/t/README
@@ -620,6 +620,13 @@ and password you'll be able to do:
 
     SMOKE_USERNAME=<username> SMOKE_PASSWORD=<password> make smoke_report
 
+You can also add an additional comment to attach to the report, and/or
+a comma separated list of tags:
+
+    SMOKE_USERNAME=<username> SMOKE_PASSWORD=<password> \
+        SMOKE_COMMENT=<comment> SMOKE_TAGS=<tags> \
+        make smoke_report
+
 Once the report is uploaded it'll be made available at
 http://smoke.git.nix.is, here's an overview of Recent Smoke Reports
 for Git:
-- 
1.7.1

